kiva/agg/build.bat

```diff
@@ -1,7 +0,0 @@
-rmdir /s /q build\temp.win32-2.4\Release
```

Comment:
I guess we don't need this file because we don't have a separate kiva.agg setup.py file?

Reply:
Yes. All building is handled by the setup.py script.